crypto/src/lib.rs

```diff
@@ -133,18 +133,21 @@ pub struct KeyGenConfiguration {
 
 impl KeyGenConfiguration {
     /// Use seed
+    #[must_use]
```

Comment:
You can also mark structures as `must_use` if it makes sense. I think an example is `Result`

Reply:
And traits. I'm trying not to mark anything that wasn't suggested by the new lints.